homeassistant/components/ruuvi_gateway/config_flow.py

```diff
@@ -0,0 +1,82 @@
+"""Config flow for Ruuvi Gateway integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import aioruuvigateway.api as gw_api
+from aioruuvigateway.excs import CannotConnect, InvalidAuth
+
+from homeassistant import config_entries
+from homeassistant.components import dhcp
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.httpx_client import get_async_client
+
+from . import DOMAIN
+from .schemata import CONFIG_SCHEMA, get_config_schema_with_default_host
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Ruuvi Gateway."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        super().__init__()
+        self.config_schema = CONFIG_SCHEMA
```

Comment:
Please don't make this a bound var since it's a constant

Reply:
It can be overwritten by the DHCP flow.